src/Request/ArraySerializer.php

```diff
@@ -23,6 +23,15 @@ final class ArraySerializer
 {
     /**
      * Serialize a request message to an array.
+     *
+     * @return array{
```

Comment:
Do we use this kind of array somewhere?
If so, I would prefer having a `psalm-type` so we can import that array shape for further usage.

Reply:
We don't currently use this